### DISCUSSION TITLE
Modular installation directory for multiple Legendary setups

### DISCUSSION THREAD

**Discussion by ghost** [legendary-gl/legendary#238] [Ideas]
Hello! I'm Seizhu and I just discovered Legendary! It's really useful and way faster than the EGL! I was toying with it around and noticed that the EG account you were logged in on the EGL didn't matter, thus you could setup multiple accounts if it was possible to maintain multiple legendary instances at the same time! It would be really helpful for the few dumbasses like me who created 3 EG accounts with different games and cosmetics on each one! 

TL;DR: Noticed the EGL login doesn't affect Legendary so maybe if we could have multiple installations we could handle multiple accounts and have stuff like this:
![legendary ideas](https://user-images.githubusercontent.com/72249264/107196864-783a7b00-69f3-11eb-943b-655c2e8e280c.png) 
to make it easier to switch accounts! 

Thanks for reading, Seizhu OUT!

**Comment by derrod** [comment]
It's not something I plan on doing, but you could override the configuration directory that Legendary uses via the `XDG_CONFIG_HOME` environment variable. That way you could have several separate configurations that can be switched between.
